e2e-tests/spring-rest-openapi-v3/src/test/kotlin/org/evomaster/e2etests/spring/openapi/v3/wiremock/inet/InetReplacementTest.kt

```diff
@@ -0,0 +1,48 @@
+package org.evomaster.e2etests.spring.openapi.v3.wiremock.inet
+
+import com.foo.rest.examples.spring.openapi.v3.wiremock.inet.InetReplacementController
+import org.evomaster.core.EMConfig
+import org.evomaster.core.problem.rest.HttpVerb
+import org.evomaster.e2etests.spring.openapi.v3.SpringTestBase
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.BeforeAll
+import org.junit.jupiter.api.Test
+
+class InetReplacementTest : SpringTestBase() {
+
+    companion object {
+        @BeforeAll
+        @JvmStatic
+        fun init() {
+            val config = EMConfig()
+            config.instrumentMR_NET = true
+            initClass(InetReplacementController(), config)
+        }
+    }
+
+    @Test
+    fun testRunEM() {
+        runTestHandlingFlakyAndCompilation(
+            "InetReplacementEM",
+            "org.foo.InetReplacementEM",
+            1000,
+            false,
```

Comment:
update `false` to `!CIUtils.isRunningGA()`
once we fix test generation for wiremock, we need to re-enable test case generation for them